test/NovaVaultV2.t.sol

```diff
@@ -297,4 +297,92 @@ contract NovaVaultV2Test is Test {
         assert(successWithdraw);
         assertEq(underlyingAmount, IERC20(underlyingAddress).balanceOf(alice));
     }
+
+    function testNovaVaultV2DoubleDeposit() public {
```

Comment:
you should start this test by transferring USDT to Alice. that part is currently missing